### PR TITLE
Sabrinawork

### PR DESCRIPTION
**Pull request by sabdj2020**
player status

### CODE REVIEW

File: mysite/checkers/consumers.py [hunk 1]
```diff
@@ -69,33 +69,39 @@ def receive(self, text_data):
         text_data_json = json.loads(text_data)
         message = text_data_json['message']
 		#below line check if click is coming from correct person or not
-        if (games[self.game_id].turn == 'D' and self.auth_user==games[self.game_id].player1) or (games[self.game_id].turn == 'L' and self.auth_user==games[self.game_id].player2):
-            if message != [-1,-1]:
-                games[self.game_id].update_game_object(message)
+        if (games[self.game_id].turn == 'D' and self.auth_user==games[self.game_id].player1) or 1 or (games[self.game_id].turn == 'L' and self.auth_user==games[self.game_id].player2):
```
Comment (zpatel-dev) [inline mysite/checkers/consumers.py]: remove this or 1

File: mysite/checkers/views.py [hunk 2]
```diff
@@ -94,48 +94,55 @@ def get(self, request):
 
 class player_statsview(View):
     def get(self, request):
-        logger.info("routing to stats page")
-        return render(request, 'checkers/player_stats.html')
+        player = str(get_current_authenticated_user())
+        #winner = Game_Session.objects.raw('select * from checkers_game_session where  player1_username="'+player+'" or player2_username="'+player+'"')
```
Comment (brucker3) [inline mysite/checkers/views.py]: why are we pushing this code if its commeted out

File: mysite/templates/checkers/player_stats.html [hunk 3]
```diff
@@ -31,6 +31,24 @@
...
+
+
+<div class="container">
+<div class="card">
+  <div class="card-header">
+    <b>Game number:<span style="color:green"><i>{{winner.game.game_id}}</i></span></b> <br>Created by: <b><i>{{winner.game.player1_username}}</i></b>
+  </div>
+  <div class="card-body">
+    <h5 class="card-title">Played against <b><i>{{winner.game.player2_username}}</i></b></h5>
+    <p class="card-text"><b><span style="color:red"><i>Winner:{{winner.winner_user}}</i></span></b></p>
```
Comment (zpatel-dev) [inline mysite/templates/checkers/player_stats.html:45]: Can you make this part better, because it is currently showing winner in red. (just a suggestion)

**Review by zpatel-dev — COMMENTED** [review]
SO far look good but . everybody have to recreate there db schema in order to run this

**Comment by zpatel-dev** [comment]
I coundt find any other error.
 in order to run this, we have to delete schema and create it again

**Review by brucker3 — APPROVED** [review]

**Pull request merged**